internal/test/e2e/grpc/verify.bats

```diff
@@ -57,8 +67,8 @@ SCOPE="go.opentelemetry.io/auto/google.golang.org/grpc"
   server_trace_id=$(server_spans_from_scope_named ${SCOPE} | jq ".traceId" | jq -Rn '[inputs]' | jq -r .[0])
   assert_equal "$server_trace_id" "$client_trace_id"
 
-  client_trace_id=$(client_spans_from_scope_named ${SCOPE} | jq ".traceId" | jq -Rn '[inputs]' | jq -r .[0])
-  server_trace_id=$(server_spans_from_scope_named ${SCOPE} | jq ".traceId" | jq -Rn '[inputs]' | jq -r .[0])
+  client_trace_id=$(client_spans_from_scope_named ${SCOPE} | jq ".traceId" | jq -Rn '[inputs]' | jq -r .[1])
+  server_trace_id=$(server_spans_from_scope_named ${SCOPE} | jq ".traceId" | jq -Rn '[inputs]' | jq -r .[1])
```

Comment:
Why did this change?

Reply:
I noticed this test was wrong while I was adding the new ones. This is supposed to check the first and second spans. The lines directly above this are already checking `[0]`, so this was supposed to be `[1]`. Figured I'd fix it here